dffml/model/slr.py

```diff
@@ -131,10 +131,14 @@ class SLRModel(SimpleModel):
                 "last_updated": "2020-11-15T16:22:25Z",
                 "prediction": {
                     "ans": {
-                        "confidence": 0.9355670103092784,
                         "value": 1
                     }
                 }
+                "confidence": {
+                    "ans": {
+                        "confidence": 0.9355670103092784,
```

Comment:
I know I just posted the recording, but I want to confirm with you that the plan here is to have the following output when this PR is not WIP:

```
                "prediction": {
                    "ans": 1
                }
                "confidence": {
                    "ans": 0.9355670103092784
                }
```

Reply:
@pdxjohnny Ah! Tbh I thought we wanted to keep it so I made the whole PR around keeping the format :facepalm: . And I had actually completed the PR lol. But yeah this makes sense. I'll fix this asap.
Also about the recording, I made the csv work with the headers, or so it seems. Please review that part as well and let me know if we still need to have that property with predictions and confidences.

Reply:
Also, we don't need the dicts inside `confidence` and `predictions` right? we could just store single values as in the above expected output you shared.

Reply:
Also would we want to have the `record.predictions()` output something similar to whats stored inside records? As there are two different outputs, one of the actual record and the other of `record.predictions()` which also gets called from other files. So that would mean `record.predictions()` output:

> I know I just posted the recording, but I want to confirm with you that the plan here is to have the following output when this PR is not WIP:
> 
> ```
>                 "prediction": {
>                     "ans": 1
>                 }
>                 "confidence": {
>                     "ans": 0.9355670103092784
>                 }
> ```

Reply:
> Also, we don't need the dicts inside `confidence` and `predictions` right? we could just store single values as in the above expected output you shared.

Yes

Reply:
> Also would we want to have the `record.predictions()` output something similar to whats stored inside records? As there are two different outputs, one of the actual record and the other of `record.predictions()` which also gets called from other files. So that would mean `record.predictions()` output:
> 
> > I know I just posted the recording, but I want to confirm with you that the plan here is to have the following output when this PR is not WIP:
> > ```
> >                 "predictions": {
> >                     "ans": 1
> >                 }
> >                 "confidences": {
> >                     "ans": 0.9355670103092784
> >                 }
> > ```


Here's what it should look like from a REPL when the PR is over

```python
>>> record.predictions()
{'ans': 1}
```


```python
>>> record.confidences()
{'ans': 0.9355670103092784}
```

> >                 }
> >                 "confidences": {
> >                     "ans": 0.9355670103092784
> >                 }